src/store/removeLiquidity.ts

```diff
@@ -122,16 +122,16 @@ const mutations = {
   },
   [types.GET_LIQUIDITY_FAILURE] (state, error) {
   },
-  [types.SET_REMOVE_PART] (state, removePart) {
+  [types.SET_REMOVE_PART] (state, removePart = 0) {
```

Comment:
Maybe it's better to use type instead of value (like: `removePart: number`) here and on the next lines?